Grids: Update Columns Sizing Topic to Add Info (#8407)
DevExpress/devextreme-documentation: concepts/05 UI Components/TreeList/10 Columns/20 Column Sizing.md

> TreeList columns have equal width in default configuration (column **width** values are *"auto"*). Total column width matches the component/container width. You can configure the following properties to change the column layout:

Reviewer: ```suggestion
TreeList columns have equal widths in the default configuration (**width** is set to *"auto"*). The total column width matches the component (container) width. Configure the following properties to change the column layout:
```

            :column-min-width="100">
            <DxColumn data-field="Title" :width="200" />
            <DxColumn data-field="Address" :min-width="150" />
            <!-- ... -->
        </DxTreeList>
    </template>

    <script>
    import { DxTreeList, DxColumn } from 'devextreme-vue/tree-list';
    // ...

    </script>

##### React

    <!-- tab: App.tsx -->
    import { TreeList, Column } from 'devextreme-react/tree-list';

    function App() {
        return (

>             <TreeList ...

Reviewer 2: ```suggestion
            <TreeList ...
                columnMinWidth={100}
            >
```
